guides/common/modules/proc_assigning-content-view-environments-to-an-activation-key-by-ID.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Assigning_content_view_environments_to_an_activation_key_by_ID{context}"]
+= Assigning content view environments to an activation key by ID
+
+To assign content view environments to an activation key, determine the IDs of the content view environments you want to assign.
+
+.CLI procedure
+
+. Run the `hammer content-view-environment list` command and note the IDs you want to use.
+. Run the following command to assign the selected content view environments to an activation key by passing their IDs as an array:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer activation-key update --organization "My_Organization" --content-view-environment-ids [1,2,3]
```

Comment:
The example values `My_Organization` and `[1,2,3]` should be italicized to indicate that they're not literal, right?

Reply:
Yes. Fixing that.